frontend/components/issues/js/issue.component.ts

```diff
@@ -1290,15 +1269,18 @@ class IssueController implements ng.IController {
 	 * Set the content height
 	 */
 	public setContentHeight() {
-		let i, length,
-			newIssueHeight = 305,
-			descriptionTextHeight = 80,
-			commentTextHeight = 80,
-			commentImageHeight = 170,
-			additionalInfoHeight = 160,
-			thumbnailHeight = 180,
-			issueMinHeight = 370,
-			height = issueMinHeight;
+
+		const newIssueHeight = 305;
+		const descriptionTextHeight = 80;
+		const commentTextHeight = 80;
+		const commentImageHeight = 170;
+		const additionalInfoHeight = 160;
+		const thumbnailHeight = 180;
+		const issueMinHeight = 370;
+
+		let height = issueMinHeight;
+		let i;
```

Comment:
Do we really want to declare `i` here? I realise it is here for legacy reasons, but should it be declared when it is needed?